src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -83,6 +84,12 @@ export class AddressForm extends React.Component<
     )
   }
 
+  getPhoneNumberInputDescription = () => {
+    if (!(this.props.billing && this.props.showPhoneNumberInput)) {
+      return "Required for shipping logistics"
+    }
+  }
```

Comment:
This logic is a bit confusing: I'm reading the body of the function as 'if NOT (`billing` AND `showPhoneNumberInput`) return 'required....', else return nothing.

Other ways we could state this:
- `'if NOT `billing` OR NOT `showPhoneNumberInput` return 'required...' else return nothing
- `if `billing AND showPhoneNumberInput` return nothing, else return 'required'

Assuming this logic is correct I prefer the final case personally.

which also feels confusing to me.

Reply:
Updated this to hopefully read a bit clearer. It's still a bit awkward though. I wish this could be more like the following but trying to avoid other changes to this components API for now.

```typescript
  phoneNumberInputDescription = (): string | null => {
    if (this.props.shipping) {
      return "Required for shipping logistics"
    } else {
      return null
    }
  }
```